<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>-10</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>STUDENT PERSONAL DATA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Other names</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineOthernames">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>70</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Surname</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineSurname">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Program Level</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxSession">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>130</y>
        <width>181</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Department</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxDepartment">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>161</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>130</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Session</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>621</width>
        <height>331</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>0</y>
         <width>391</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ACADEMIC QUALIFICATION</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Exam Number</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineExamNumber">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>171</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>70</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Exam Number</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineExamNumber2">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>70</y>
         <width>171</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>20</width>
         <height>241</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Exam Type</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxExamType">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Subjects Offered</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>171</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_2">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>130</y>
         <width>171</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>100</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Exam Type</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxExamType2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>130</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Year</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxYear">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>170</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Year</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxYear2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>190</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAddSubject">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Subject</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAddSubject2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>250</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Subject</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>100</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Subjects Offered</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxTwoSittings">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>96</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>2 sittings</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonRemove">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>95</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonRemove2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>280</y>
         <width>95</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButtonSave">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>540</y>
        <width>95</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>540</y>
        <width>95</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Clear</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxLevel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>161</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>DOB</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>App. Number</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineAppNumber">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Sex</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxSex">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>100</y>
        <width>181</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineAddress">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateDOB">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>160</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>State</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxState">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>40</y>
        <width>181</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Marital Status</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxMaritalStatus">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>161</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>190</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Religion</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxReligion">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>190</y>
        <width>181</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineAppNumber</tabstop>
  <tabstop>lineSurname</tabstop>
  <tabstop>lineOthernames</tabstop>
  <tabstop>comboBoxState</tabstop>
  <tabstop>lineAddress</tabstop>
  <tabstop>comboBoxSex</tabstop>
  <tabstop>comboBoxLevel</tabstop>
  <tabstop>comboBoxSession</tabstop>
  <tabstop>comboBoxDepartment</tabstop>
  <tabstop>dateDOB</tabstop>
  <tabstop>comboBoxMaritalStatus</tabstop>
  <tabstop>comboBoxReligion</tabstop>
  <tabstop>lineExamNumber</tabstop>
  <tabstop>comboBoxExamType</tabstop>
  <tabstop>spinBoxYear</tabstop>
  <tabstop>pushButtonAddSubject</tabstop>
  <tabstop>pushButtonRemove</tabstop>
  <tabstop>lineExamNumber2</tabstop>
  <tabstop>comboBoxExamType2</tabstop>
  <tabstop>spinBoxYear2</tabstop>
  <tabstop>pushButtonAddSubject2</tabstop>
  <tabstop>pushButtonRemove2</tabstop>
  <tabstop>checkBoxTwoSittings</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>listWidget_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
